pages/index.js

```diff
@@ -1,6 +1,6 @@
-import Head from 'next/head'
```

Comment:
Can you check if these changes are still imposed by `prettier`? We have changed our config so these might need to be reverted.